src/pybind/mgr/cephadm/services/monitoring.py

```diff
@@ -61,7 +61,10 @@ def generate_config(self, daemon_spec: CephadmDaemonSpec) -> Tuple[Dict[str, Any
 
     def get_active_daemon(self, daemon_descrs: List[DaemonDescription]) -> DaemonDescription:
         # Use the least-created one as the active daemon
-        return daemon_descrs[-1]
+        if len(daemon_descrs) > 0:
```

Comment:
fyi, this is the same as 

```suggestion
        if daemon_descrs:
```